### PR TITLE
Improvements 06

### PR DESCRIPTION
**Pull request by MatthewCroughan**

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 7c98638cc2ecec85db89c89e48030615c833856f are browesable at: https://ci.foundries.io/projects/fio-docs/builds/758/docs/artifacts/html/index.html

File: source/getting-started/signup/index.rst [hunk 1]
```diff
@@ -26,6 +26,11 @@ When your account is created, it is not associated with any factories.
 
 Create one by clicking :guilabel:`Create Factory`
 
+.. warning:: 
+
+   Once a Factory is created, it cannot be renamed without contacting
+   support at http://support.foundries.io/
```
Comment (doanac) [inline source/getting-started/signup/index.rst:32]: I'm not even sure we can say that. We aren't "renaming" them. We just create a new factory.

Comment (MatthewCroughan) [reply source/getting-started/signup/index.rst]: Is this a better solution? https://github.com/foundriesio/docs/pull/168/commits/8b81ffaec62b86000395cf4f24b5799f363700ac

**Comment by doanac** [comment]
Docs for adfd0c9aaa9e004b733f18d58114d9a2ee058fc1 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/759/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 9f288ebb4f89f867f7cfa15236e412de61a46d00 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/760/docs/artifacts/html/index.html

**Pull request merged**